protocol-configuration/mainnetAddresses.ts

```diff
@@ -773,49 +773,52 @@ const MainnetContractsConfig: MainnetContractsConfig = {
   aaveTribeIncentivesController: {
     artifactName: 'IAaveIncentivesController',
     address: '0xDee5c1662bBfF8f80f7c572D8091BF251b3B0dAB',
-    category: AddressCategory.Rewards
+    category: AddressCategory.Deprecated
   },
 
   aaveTribeIncentivesControllerImpl: {
```

Comment:
Seems not needed, no?

Reply:
Our Aave Tribe rewards ended back in March 2022 and we're now ending all Tribe rewards and deprecating the sub-system that administers them. 

This category key helps us categorise and keep track of active contracts. As we're deprecating the incentives system, all associated contracts are being marked as `Deprecated`